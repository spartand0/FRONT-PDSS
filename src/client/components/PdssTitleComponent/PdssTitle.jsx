import React from 'react';

function PdssTitle({ title }) {
	return (
		<div className="cell">
			<h3>{title}</h3>
		</div>
	);
}

export default PdssTitle;
