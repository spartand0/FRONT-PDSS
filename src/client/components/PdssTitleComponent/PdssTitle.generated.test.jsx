import renderer from 'react-test-renderer';
import React from 'react';
import PdssTitle from './PdssTitle';

const renderTree = tree => renderer.create(tree);
describe('<PdssTitle>', () => {
  it('should render component', () => {
    expect(renderTree(<PdssTitle 
    />).toJSON()).toMatchSnapshot();
  });
  
});