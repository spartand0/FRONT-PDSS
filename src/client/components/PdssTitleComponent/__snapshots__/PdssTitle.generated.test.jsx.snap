// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PdssTitle> should render component 1`] = `
<div
  className="cell"
>
  <h3 />
</div>
`;
